elastic/kibana
# [Custom threshold] Save the ECS group by fields at the AAD root level

Related to #183220

## Summary

This PR saves the ECS group by fields at the AAD root level.

|Group by fields| AAD document|
|---|---|
|![image](https://github.com/user-attachments/assets/47f2b6e6-67b0-4a15-8d42-fbee4d25cafe)|![image](https://github.com/user-attachments/assets/6429648c-848e-4ece-87bb-041a90a7d2d4)|

### 🧪 How to test
- Create a custom threshold rule with multiple groups (both ECS and non-ECS fields)
- Check the related AAD document; you should be able to see the ECS fields at the root level and not seeing non-ECS fields there
- Check the same information for the recovered alerts
- Rules without group by should work as before
